docs/install/operator/configuring-eventing-cr.md

```diff
@@ -583,3 +583,20 @@ kn operator configure labels --component eventing --serviceName eventing-webhook
 kn operator configure annotations --component eventing --serviceName eventing-webhook --key myannotations --value bar -n knative-eventing
 kn operator configure selectors --component eventing --serviceName eventing-webhook --key myselector --value bar -n knative-eventing
 ```
+
+## Override system podDisruptionBudgets
+
+You can configure the `minAvailable` for a specific podDisruptionBudget resource based on the name. For example, if you would like
+to change `minAvailable` into 70% for the podDisruptionBudget named `eventing-webhook`, you need to change your KnativeEventing CR as below:
```

Comment:
>Can you explain what podDisruptionBudget is or what it does? When we don't explain these things I think the docs have limited usefulness

I agree with @abrennan89 you could link to the Kubernetes [docs](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) that explain in more details

Reply:
I think more than just adding a link, for these type of sections we should have 1-2 lines before the config as an intro paragraph explaining who would use this setting and why. Otherwise what's the use case for adding it if we can't explain that to users?

Reply:
ping @houshengbo - do you plan to do more updates to this PR?